setup.py

```diff
@@ -2,18 +2,17 @@
 
 setup(
     name='funsor',
-    version='0.0.0',
+    version='0.1.0',
```

Comment:
Would you like to populate the long description for pypi? Its totally cool if you are looking to skip it - just wanted to make sure.

Reply:
Good point, I'll add that.

Reply:
done.